Makefile

```diff
@@ -0,0 +1,33 @@
+DOCKER_IMAGE = oam/server:latest
+
+all: server
+
+server:
+	@docker build -f ./server/Dockerfile -t $(DOCKER_IMAGE) ./server
+
+start: server
+	@docker run \
+		--detach \
+		--name oam-server \
+		--publish 8000:8000 \
+		--volume $(PWD)/server:/app \
+		$(DOCKER_IMAGE) start
+
+test: start
+	@sleep 1
+
+	@docker run \
+		--rm \
+		--name oam-server-test \
+		--link oam-server:oam-server \
+		--volume $(PWD)/server:/app \
```

Comment:
`link` in my sentence meant to "link" (mount) the folders, sorry for the wording.
I updated my PR ( #23) to use symlinks in a way that allows the user to do local installs and keeps folder weirdness inside the repo directory.